README.md

```diff
@@ -51,5 +51,5 @@ Now use it in anywhere in your HTML or Markdown.
 
 - `npm start`: Main dev server. Runs everything you need. 
 - `npm run dev`: Runs components in isolation. Serves `app/index.html` as a playground for components. 
-- `npm run hugo`: Only runs static site. 
-- `npm run build`: Build for production
+- `npm run hugo`: Only runs static sites. 
```

Comment:
using the `site` keyword is correct because here we are talking about the current repo project to `run`